packages/ui-server/routes/api/_entry.js

```diff
@@ -0,0 +1,70 @@
+const webpackEntryPoint = {};
+function forEachPromise(entries, fn) {
+	return entries.reduce((promise, prop) => {
+		const trimmedProp = prop.trim();
+
+		return promise.then((n) => {
+			if (n) {
+				Object.keys(n).forEach((val) => {
+					if (
+						n[val].charAt(0) !== "(" &&
+						n[val].charAt(0) !== "[" &&
+						!n[val].includes("function") &&
+						!n[val].includes("path") &&
+						!n[val].includes("process")
+					) {
+						n[val] = `\'${n[val].replace(/"|'/g, "").concat(".js")}\'`;
+					}
+					webpackEntryPoint[val] = n[val];
+				});
+			} else {
+				n = {};
+			}
+			return fn(trimmedProp);
+		});
+	}, Promise.resolve());
+}
+
+module.exports = function(questioner, answer) {
+	if (answer === true) {
+		return questioner.question({
+			action: "question",
+			question: "Type the names you want for your entry files",
+			type: "array"
+		}).then((entries) => {
+			return forEachPromise(entries, (entryProp) => {
+				return questioner.question({
+					action: "question",
+					question: `What is the location of "${entryProp}"? [example: ./src/${entryProp}]`
+				});
+			}).then((entryPropAnswer) => {
+				Object.keys(entryPropAnswer).forEach(val => {
+					if (
+						entryPropAnswer[val].charAt(0) !== "(" &&
+						entryPropAnswer[val].charAt(0) !== "[" &&
+						!entryPropAnswer[val].includes("function") &&
+						!entryPropAnswer[val].includes("path") &&
+						!entryPropAnswer[val].includes("process")
+					) {
+						entryPropAnswer[val] = `\'${entryPropAnswer[val]
+							.replace(/"|'/g, "")
+							.concat(".js")}\'`;
+					}
+					webpackEntryPoint[val] = entryPropAnswer[val];
+				});
+				return webpackEntryPoint;
+			});
+		});
+	} else {
+		return questioner.question({
+			action: "question",
+			question: "Which module will be the first to enter the application?"
+		}).then((singularEntryAnswer) => {
+			let singularEntry = singularEntryAnswer;
+			if (singularEntry.indexOf("\"") >= 0) {
+				singularEntry = singularEntry.replace(/"/g, "'");
+			}
+			return singularEntry;
```

Comment:
@rishabh3112 we could reduce this to:

```
return singularEntryAnswer.indexOf("\"") >= 0
  ? singularEntryAnswer.replace(/"/g, "'")
  : singularEntryAnswer;
```

Reply:
Yeah this can be done, but I want to maintain as much similarity as possible with the prompting of init-generator so it is easier to manipulate here when munipulated at init-generator.

Reply:
@rishabh3112 why do not change the `init-generator` too?

Reply:
Not there is use case on checking input length else use the default value. This case isn't valid here as this must be handled on client side for UI.

Reply:
@rishabh3112 sorry I didn't get your point. Here is just a readability and code style change, it won't change nothing in the way of the code works.

Reply:
Have a look at init-generator, you would get what I want to say here 👍